src/Sulu/Bundle/ContentBundle/Collaboration/Collaboration.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContentBundle\Collaboration;
+
+/**
+ * Represents a user, which is currently editing an entity.
+ */
+class Collaboration
```

Comment:
Actually I would tend to rename the `ResourceBundle` to `FilterBundle`, since this functionality is already quite complex on its own. In general I would prefer to have smaller bundles, since that would also make it easier to enable or disable features, without struggling around with quite complex configuration and extension classes, or even compiler passes.